adba/aniDBAbstracter.py

```diff
@@ -80,7 +81,7 @@ def _easy_extend(initialList, item):
         if item:
             if isinstance(item, list):
                 initialList.extend(item)
-            elif isinstance(item, str):
+            elif isinstance(item, string_types):
```

Comment:
Prefer to limit to `text_type` if possible.

Reply:
I don't know. What if item is a binary type? I'm not confident with this code, to assume it's always unicode.